### DISCUSSION TITLE
Next Version

### DISCUSSION THREAD

**Discussion by suhanth94** [LogNet/grpc-spring-boot-starter#334] [Q&A]
Hi, this library is internally using grpc-netty-shaded JAR which has vulnerabilities . When can we expect these fixed and the next version be released?

**Comment by jvmlet** [comment]
Are you after the build with latest grpc version `1.52.1` ?

**Reply by jvmlet** [reply]
Can you please post the link to the vulnurability in `grpc-netty-shaded`?

**Reply by suhanth94** [reply]
https://nvd.nist.gov/vuln/detail/cve-2017-12613
https://nvd.nist.gov/vuln/detail/CVE-2021-35940
https://nvd.nist.gov/vuln/detail/CVE-2022-41881
https://nvd.nist.gov/vuln/detail/CVE-2022-41915

**Reply by suhanth94** [reply]
I expect  some of these to be resolved if we update grpc version to 1.52.1

**Reply by suhanth94** [reply]
@jvmlet can you please help on the above query? We are waiting for the new version.

**Reply by jvmlet** [reply]
https://github.com/LogNet/grpc-spring-boot-starter/blob/master/ReleaseNotes.md

**Reply by suhanth94** [reply]
@jvmlet we are still using java 11 in our application. are we not releasing compatible versions for older java versions?
